src/clients/HttpTunneler.cc

```diff
@@ -59,14 +61,12 @@ Http::Tunneler::doneAll() const
     return !callback || (requestWritten && tunnelEstablished);
 }
 
+// TODO: Remove this diff-reducer before merging this PR.
```

Comment:
This and other "before merging" TODOs should and will be addressed before merging the PR. Please do not block this PR on their account. Once these diff reducers are removed, the diff noise will increase, complicating comprehension of the important/key PR changes.

Reply:
Done in 146c808.